flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/metadata/FlinkRelMdModifiedMonotonicityTest.scala

```diff
@@ -340,8 +340,7 @@ class FlinkRelMdModifiedMonotonicityTest extends FlinkRelMdHandlerTestBase {
   def testGetRelMonotonicityOnChangelogNormalize(): Unit = {
     assertEquals(
       new RelModifiedMonotonicity(Array(
-        CONSTANT, CONSTANT, NOT_MONOTONIC, NOT_MONOTONIC,
-        NOT_MONOTONIC, NOT_MONOTONIC, NOT_MONOTONIC)),
+        CONSTANT, CONSTANT, NOT_MONOTONIC, NOT_MONOTONIC)),
```

Comment:
Why needs do this?

Reply:
Previously, we constructed `FlinkRelMdHandlerTestBase`#`changelogNormalize` by using  `FlinkRelMdHandlerTestBase`#`studentStreamScan`  as its `TableScan`, which I cannot get `TableSourceTable` from  `studentStreamScan` cuz `RelOptTable` in `studentStreamScan`  is a instance of `MockMetaTable`. So I used `FlinkRelMdHandlerTestBase`#`tableSourceTableStreamScan` instead of  `studentStreamScan` now., And it seems that **only this test was affected**.

Reply:
Thank you for the explanation